src/webview/openshift-terminal/openShiftTerminal.ts

```diff
@@ -461,6 +485,9 @@ export class OpenShiftTerminalManager implements WebviewViewProvider {
                     } else if (message.kind === 'resize') {
                         terminal.resize(message.data.cols, message.data.rows);
                     } else if (message.kind === 'closeTerminal') {
+                        if (terminal.file.endsWith('func.exe')) {
```

Comment:
1. Please use the `onExit` callback, eg. pass these three lines of code as a lambda function here: https://github.com/redhat-developer/vscode-openshift-tools/pull/3843/files#diff-f9b7ce8b2e1d73c6d56dbcb8003e652267411f5c89e851753523d568b575fc48R121
2. This only works on Windows, since on Linux and macOS the file name is `func` (without `.exe`). However, if you use the callback method described in 1., you should be able to avoid making this check, since the callback will only be called in the `func`-related commands.

Reply:
**Point 1** didn't work it out, `onExit` will be called when the process terminated. But I want to refresh the tree when the terminal was closed.

**Point 2**, I will update the check based on the platform.